pkg/authhelpers/gcpauth_test.go

```diff
@@ -0,0 +1,73 @@
+package authhelpers
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestResolveGoogleCredentials_ValidCredentials(t *testing.T) {
+	t.Parallel()
```

Comment:
As this mutates the environment it can't be a parallel test. Add `
//nolint:paralleltest // mutates environment variables` above to keep the linter happy.